src/parser.rs

```diff
@@ -432,14 +435,14 @@ fn parse_unspec_value<'a>(
             map(take_while1(is_safe_unquoted_chr), |s| {
                 Value::Str(s, Quote::None)
             }),
-            peek(take_while1(is_sep)),
+            peek(take_while(is_sep)),
```

Comment:
I don't know much about nom... Now, as the inputs can have no newline at the end (and therefore, it would be an EOF), we can't test the last character with `take_while1(is_sep)` as it would raise an EOF error. The example `testdata/line-daemon-end-2.txt"` failed parsing if `take_while1(is_sep)` is used instead of `take_while(is_sep)`. 

I don't know if this change is ok or if we should change more things because we can have EOF at the end.. What do you think?